packages/core/src/api/exceptions/types.ts

```diff
@@ -20,7 +19,9 @@ export interface Stacktrace {
   frames: ExceptionStackFrame[];
 }
 
-export type ExceptionContext = BaseObject;
+export type ExceptionContext = {
```

Comment:
most idiomatic TS way would be `export type ExceptionContext = Record<string, string>`

Reply:
Oh yes of course thank